src/api/lib/backend/test.rb

```diff
@@ -0,0 +1,47 @@
+# API for accessing to the backend
+module Backend
+  class Test
+    @backend = nil
+
+    # Starts the test backend
+    def self.start(wait_for_scheduler = false)
```

Comment:
Yes, sure, I was also thinking about something like that. I will change it. Thanks